Model version policy, Tensorflow, functional tests (#20)
IntelAI/OpenVINO-model-server: docs/docker_container.md

**Note:** Dynamic batch size _is not_ supported.

Processing bigger batches of requests increases the throughput but the side effect is higher latency.

## Model Version Policy

> Model version policy let to decide with versions of model has to be served by OVMS. this parameter allows you to control the memory consumption of the server and 

Reviewer: ```suggestion
Model version policy makes it possible to decide which versions of model will be served by OVMS. This parameter allows you to control the memory consumption of the server and 
```

> decide which version will be used regardless of what is located under the path given when the server is started.

Reviewer: ```suggestion
decide which versions will be used regardless of what is located under the path given when the server is started.
```
